examples/assets/scripts/camera-frame.mjs

```diff
@@ -0,0 +1,497 @@
+// Camera Frame v 1.0
```

Comment:
Can't we pull this script from the 2.5.0 NPM package? Or is this modified somehow from that?

Reply:
Yeah we could do. One thing,  you could resolve module paths in the pc-assets using the import map. eg use `src="playcanvas/scripts/esm/camera-frame.mjs"` and use `import.resolve.meta(url)` which should return the right path

Reply:
but otherwise can just load from a cdn etc

Reply:
Other examples do this:

https://github.com/playcanvas/web-components/blob/e3ff214fe530a825251333fec940f0cdbdf494c8/examples/animation.html#L20-L22